### ISSUE TITLE
Create page on transferring/running code

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/ev3dev.github.io#271]
As a basic measure, I think we should have a page that explains how to copy code to the brick and run it in generic terms. Probably using FileZilla or another similar app. We should link to it from Getting Started, among other places.

**Comment by bmegli** [comment]
Or we could have a section in tutorials (or just options in one page) for various ways of doing it:
- FileZilla or similiar GUI app
- `scp`/`pscp` for doing it from command line 
- `sftp`/`psftp` for handy transfer of multiple files interactively from the command line
- filesharing

Doing it with `scp` may be important for automating some tasks(like post build in IDE)
`sftp` is handy if you are working command line/to command line (host to EV3)

For running also:
- from Brickman
- `ssh`/`plink`

Doing it from 'ssh`/`plink` may be again imortant for automating some tasks (like running the program from host directly from IDE).

**Comment by WasabiFan** [comment]
Yeah, that makes sense. I think we will need o start small with a simple GUI program, but we will definitely want to expand on that in the future.
